packages/contracts-bedrock/test/kontrol/README.md

```diff
@@ -0,0 +1,88 @@
+# Kontrol Verification
+
+This folder contains Kontrol symbolic property tests.
+
+## Directory structure
+
+The directory is structured as follows
+
+```tree
+test/kontrol
+├── KontrolDeployment.sol
+├── pausability-lemmas.k
+├── proofs
+│   ├── interfaces
+│   │   └── KontrolInterfaces.sol
+│   ├── OptimismPortal.k.sol
+│   ├── tests
+│   │   ├── Counter.sol
+│   │   └── Counter.t.sol
+│   └── utils
+│       ├── DeploymentSummaryCode.sol
+│       ├── DeploymentSummary.sol
+│       └── KontrolUtils.sol
+├── README.md
+└── scripts
+    ├── json
+    │   ├── clean_json.py
+    │   └── reverse_key_values.py
+    ├── make-summary-deployment.sh
+    └── run-kontrol.sh
+```
+
+### Root folder
+
+- [`KontrolDeployment.sol`](./KontrolDeployment.sol): Reduced deployment to generate the summary contract
+- [`pausability-lemmas.k`](./pausability-lemmas.k): File containing the necessary lemmas for this project
+- [`proofs`](./proofs): Where the proofs of the project live
+- [`scripts`](./scripts): Where the scripts of the projects live
+
+### [`proofs`](./proofs) folder
+
+- [`OptimismPortal.k.sol`](./proofs/OptimismPortal.k.sol): Symbolic property tests
+- [`interfaces`](./proofs/interfaces): Files with the signature of the functions involved in the verification effort
+- [`utils`](./proofs/utils): Dependencies for `OptimismPortal.k.sol`, including the summary contracts
+
+### [`scripts`](./scripts) folder
+
+- [`make-summary-deployment.sh`](./scripts/make-summary-deployment.sh): Executes [`KontrolDeployment.sol`](./KontrolDeployment.sol), curates the result and writes the summary deployment contract
+- [`run-kontrol.sh`](./scrpts/run-kontrol.sh): Proof execution script
+- [`json`](./scripts/json): Data cleaning scripts for the output of [`KontrolDeployment.sol`](./KontrolDeployment.sol)
+
+## Verification exeuction
+
+The verification execution consists of two steps, although the first step may be ommited to use the commited version. There's one script to run per step. These scripts should be run from the [`contracts-bedrock`](../../) directory.
+
+1. Generate a deployment summary contract from [`KontrolDeployment.sol`](./KontrolDeployment.sol)
+```bash
+  ./test/kontrol/scripts/make-summary-deployment.sh
```

Comment:
When I run this I get an error at the very end of `kontrol: error: unrecognized arguments: --contract-names --output-dir test/kontrol/proofs/utils`

More context from the logs below:

```
Created state diff json
Operation completed. Result saved to snapshots/state-diff/Kontrol-Deploy.json
Cleaned state diff json
Reversed JSON saved to deployments/hardhat/.deployReversed
usage: kontrol [-h] {version,compile,solc-to-k,build,summary,prove,show,to-dot,list,view-kcfg,remove-node,simplify-node,step-node,merge-nodes,section-edge,get-model} ...
kontrol: error: unrecognized arguments: --contract-names --output-dir test/kontrol/proofs/utils
```